app/assets/javascripts/times.js

```diff
@@ -0,0 +1,130 @@
+/* global $ */
+
+/* Datepicker */
+$(document).on('turbolinks:load', 
+    function (){
+        $('.datepicker').datepicker({todayHighlight: true, multidate: true, format: 'yyyy-mm-dd'});
+    }
+);
+
+/* Show highlighted datepicker dates */
+$(document).on('turbolinks:load',
+    function(){
+        $('.datepicker').datepicker().on('changeDate', function(e){
+            var string = $('.datepicker').val();
+            var dateArray = string.split(",");
+            var formatted = "";
+            
+            for (var i = 0; i < dateArray.length && string.length > 0; i++){
+                var unformatted = dateArray[i];
+                var niceFormat = formatDate(unformatted);
+                var element = "#"+unformatted;
+                if ($(element).length == 0){
+                    addDiv(unformatted);
+                }
+                formatted += niceFormat + '\n';
+            }
+            
```

Comment:
When displaying the chosen dates on the right hand side, I think that it would be better if the dates were sorted so that it's a bit more user friendly.